ext/afform/core/Civi/Api4/Action/Afform/Submit.php

```diff
@@ -68,6 +68,7 @@ protected function processForm() {
     \Civi::dispatcher()->dispatch('civi.afform.validate', $event);
     $errors = $event->getErrors();
     if ($errors) {
+      \Civi::log()->error('Afform Validation errors: ' . print_r($errors, TRUE));
```

Comment:
@mattwire what about instead `\Civi::log()->error('Afform Validation Errors: :errors', [':errors' => $errors])`

Reply:
Actually - that's not the syntax - see https://docs.civicrm.org/dev/en/latest/framework/logging/ - which is based on psr3

Reply:
Hmm, so not sure what I'm supposed to do here? Are we happy with what I've put in or should I change it?